crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -1129,9 +1129,8 @@ fn handle_with_item_comment<'a>(
     if comment.end() < as_token.start() {
         // If before the `as` keyword, then it must be a trailing comment of the context expression.
         CommentPlacement::trailing(context_expr, comment)
-    }
-    // Trailing end of line comment coming after the `as` keyword`.
-    else if comment.line_position().is_end_of_line() {
+    } else if comment.line_position().is_end_of_line() {
+        // Trailing end of line comment coming after the `as` keyword`.
```

Comment:
And you claim to be unopinionated about formatting :P

Reply:
I deserve this